packages/client/src/pages/settings/preferences-registry.vue

```diff
@@ -0,0 +1,380 @@
+	// iOS Safari で正常に動かす為のおまじない
+	(window as any).__misskey_input_ref__ = input;
+
+	input.click();
+}
+
+async function applyRegistry(id: string) {
+	if (!registries) return;
+
+	const registry = registries[id];
+
+	const { canceled: cancel1 } = await os.confirm({
+		type: 'warning',
+		title: ts._preferencesRegistry.apply,
+		text: t('_preferencesRegistry.applyConfirm', { name: registry.name }),
+	});
+	if (cancel1) return;
+
+	// defaultStore
+	for (const [key, value] of Object.entries(registry.defaultStore)) {
+		if (key in defaultStore.def && defaultStore.def[key].where !== 'account') {
+			defaultStore.set(key as keyof Registry['defaultStore'], value);
+		}
+	}
+
+	// coldDeviceStorage
+	for (const [key, value] of Object.entries(registry.coldDeviceStorage)) {
+		ColdDeviceStorage.set(key as keyof Registry['coldDeviceStorage'], value);
+	}
+
+	// fontSize
+	if (registry.fontSize) {
+		localStorage.setItem('fontSize', registry.fontSize);
+	} else {
+		localStorage.removeItem('fontSize');
+	}
+
+	// useSystemFont
+	if (registry.useSystemFont) {
+		localStorage.setItem('useSystemFont', registry.useSystemFont);
+	} else {
+		localStorage.removeItem('useSystemFont');
+	}
+
+	// wallpaper
+	if (registry.wallpaper != null) {
+		localStorage.setItem('wallpaper', registry.wallpaper);
+	} else {
+		localStorage.removeItem('wallpaper');
+	}
+
+	const { canceled: cancel2 } = await os.confirm({
+		type: 'info',
+		text: ts.reloadToApplySetting,
+	});
+	if (cancel2) return;
+
+	unisonReload();
+}
+
+async function deleteRegistry(id: string) {
+	if (!registries) return;
+
+	const { canceled } = await os.confirm({
+		type: 'info',
+		title: ts._preferencesRegistry.delete,
+		text: t('_preferencesRegistry.deleteConfirm', { name: registries[id].name }),
+	});
+	if (canceled) return;
+
+	await os.apiWithDialog('i/registry/remove', { scope, key: id });
+	delete registries[id];
+}
+
+async function save(id: string) {
+	if (!registries) return;
+
+	const { name, createdAt } = registries[id];
+
+	const { canceled } = await os.confirm({
+		type: 'info',
+		title: ts._preferencesRegistry.save,
+		text: t('_preferencesRegistry.saveConfirm', { name }),
+	});
+	if (canceled) return;
+
+	const registry: Registry = {
+		name,
+		createdAt,
+		updatedAt: (new Date()).toISOString(),
+		version,
+		defaultStore: getDefaultStoreValues(),
+		coldDeviceStorage: ColdDeviceStorage.getAll(),
+		fontSize: localStorage.getItem('fontSize'),
+		useSystemFont: localStorage.getItem('useSystemFont') as 't' | null,
+		wallpaper: localStorage.getItem('wallpaper'),
+	};
+	await os.apiWithDialog('i/registry/set', { scope, key: id, value: registry });
+}
+
+async function rename(id: string) {
+	if (!registries) return;
+
+	const { canceled: cancel1, result: name } = await os.inputText({
+		title: ts._preferencesRegistry.inputName,
+	});
+	if (cancel1 || registries[id].name === name) return;
+
+	if (Object.entries(registries).some(e => e[1].name === name)) {
```

Comment:
Using `e` as an identifier is not allowed, this adds lint errors (prefer `registry` or something shorter here)